mint-auditor/src/bin/main.rs

```diff
@@ -72,6 +73,21 @@ pub enum Command {
         #[clap(long, env = "MC_JSON")]
         json: bool,
     },
+    StartHttpServer {
+        /// Path to mint auditor db.
+        #[clap(long, parse(from_os_str), env = "MC_MINT_AUDITOR_DB")]
+        mint_auditor_db: PathBuf,
+
+        /// Oprtional HTTP listen URI, to be used when HTTP API access is
```

Comment:
The comment doesn't match what this argument is. Additionally, it shouldn't be optional since it is unclear what it defaults to. Please use clap's default value to default to 127.0.0.1 (for the host) and a port of your choosing,

Reply:
woops, left over from previous refactor